### PR TITLE
Global state app load

### PR DESCRIPTION
**Pull request by breyana**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -4,13 +4,13 @@ const webpack = require('webpack')
 const determineHost = () => {
   switch ( process.env.NODE_ENV ) {
     case 'production':
-      return "'http://localhost:1337'"
+      return 'http://localhost:1337'
```
Comment (nodatall) [inline webpack.config.js]: Ok it turns out that you are right, we don't need the extra "". Keep it this way.

File: source/server/controllers/v1/appController.js [hunk 2]
```diff
@@ -5,4 +5,6 @@ const handleSendApp = ( request, response ) => {
   response.sendFile( ( `${publicPath}/index.html` ) )
 }
 
-export default handleSendApp
+export {
```
Comment (breyana) [inline source/server/controllers/v1/appController.js]: @nodatall changing this back is somehow causing errors

**Comment by breyana** [comment]
@nodatall changes committed

**Comment by breyana** [comment]
@nodatall conflicts have been resolved

**Pull request merged**